source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -318,6 +318,10 @@
             "type": "object",
             "description": "Options given to @babel/preset-env. With this options the output type of babel can be defined. For details see here: <https://babeljs.io/docs/en/babel-preset-env#options>. They can be overridden per target."
           },
+          "browserifyOptions": {
+            "type": "object",
+            "description": "Options given to browserify. For details see here: <https://github.com/browserify/browserify#usage>. They can be overridden per target."
```

Comment:
`https://github.com/browserify/browserify#usage` is not modified in this PR and I haven't seen any other PR that updates it...

Reply:
Don't understand this notice

Reply:
Disregard. I misread it.